docs/docs/Tutorials/chat-with-files.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Create a chatbot with your own files
+slug: /chat-with-files
+---
+
+import Icon from "@site/src/components/icon";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Create a chatbot application that chats with files loaded from your local machine.
+
+## Prerequisites
+
+- [A running Langflow instance](/get-started-installation)
+- [An OpenAI API key](https://platform.openai.com/api-keys)
+
+## Create a file loader chat flow
+
+1. In Langflow, click **New Flow**, and then select the **Basic prompting** template.
+2. Add your **OpenAI** API key to the **Language Model** component.
+3. To confirm that the chatbot responds as expected, open the <Icon name="Play" aria-hidden="true" /> **Playground** and then ask a question.
+The LLM should respond as the **Prompt** component specifies.
+With the basic flow responding correctly, modify the **Prompt** component to accept additional inputs.
+4. In the **Prompt** component, open the **Template** field, and replace the current prompt with the following text:
+    ```text
+    ChatInput:
+    {chat-input}
+    File:
+    {file}
+    ```
+    The **Prompt** component adds ports to accept new inputs for each value with curly braces, in this case, a chat input and a file.
+
+    :::tip
+    The port names are for illustrative purposes. You don't have to name them after the component they're connected to.
+    :::
+
+5. Add a [File component](/components-data#file) to the flow, and connect the **Raw Content** output to the Prompt component's `file` input.
+
+    Your flow should look like this:
+
+    ![File loader chat flow](/img/tutorial-chat-file-loader.png)
+
+    The flow is complete.
+    If you'd like, add files to the File component and chat with it within the Langflow IDE.
+    In the next section, you will load files and chat with your flow from a Python application.
```

Comment:
```suggestion
```

See my suggestion on like 37 to move this content closer to the line where you add the file component.